beacon_node/eth1/src/service.rs

```diff
@@ -1028,7 +1060,7 @@ impl Service {
     /// - Err(_) if there is an error.
     ///
     /// Emits logs for debugging and errors.
-    pub async fn update_block_cache(
+    pub async fn update_block_cache<'e>(
```

Comment:
As above.

Reply:
I do believe it's desirable to see the startup/initial value so I know what the initial value is, why don't you want it?

Another deseriable capability is to have an api which returns the current value, if that was or is already provided then I could be convinced the initial value isn't needed. What do you think?

Finally, if you rather get some in quickly, we can abandoned this PR and just take yours, https://github.com/paulhauner/lighthouse/commit/d917c5d7650280e02f47380fd084528d1a45a0ac. I'm fine with that!!